contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -1,34 +1,40 @@
 ---
-name: "bitcoin-linux-0.11"
+name: "zcash-1.0.0-rc1"
```

Comment:
Since the version number is hard-coded here, we could probably make use of a script that updates the version number in this file and all other similar places when running releases.